src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -17,13 +17,17 @@
  * under the License.
  */
 
-import { EuiHeaderLogo } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 import useObservable from 'react-use/lib/useObservable';
 import { Observable } from 'rxjs';
 import Url from 'url';
 import { ChromeNavLink } from '../..';
+// @ts-expect-error TS doesn't like importing an SVG
+import elasticMark from './elastic_mark.svg';
```

Comment:
:shrug: No idea why TS doesn't think this svg exists, but it does, and loads in the browser just fine. I see similar patterns in the rest of Kibana?

Reply:
TS never likes importing non-TS files... I converted it to a .tsx file for you. This way if anyone else tries to import it as well, the fill will match the location's text color.

Reply:
Awesome, thanks!